armory/art_experimental/poison_detection/spectral_signature_defense.py

```diff
@@ -10,7 +8,7 @@
         PoisonFilteringDefence,
     )
 except ImportError:
-    logger.warning(
+    log.warning(
         "ART 1.2 support is deprecated and will be removed in ARMORY 0.11. Use ART 1.3"
```

Comment:
This is not in line with the PR, but this comment looks really old, maybe consider revising?

Reply:
removed. this used compatibility libraries if import failed, but now it should just fail.